pyproject.toml

```
@@ -0,0 +1,30 @@
+[project]
+name = "vehicle-simulation-environment"
+dynamic = ["version"]
+dependencies = [
+  "numpy",
+]
+readme = "README.md"
+
+[project.optional-dependencies]
+dev = [
+  "ruff",
+  "black",
+  "pre-commit"
+]
+
+[tool.ruff.lint]
+select = [
+    # pycodestyle
+    "E",
+    # Pyflakes
+    "F",
+]
+
+[tool.setuptools.packages.find]
+where = [
+  "src"
+]
+
+[project.scripts]
+simulate-cli = "simulation:main.main"
```

```suggestion
vehicle-simulation-environment = "vehicle_simulation_environment.__main__:main"
```
For the sake of when we get this integrated into `selfdrive`, it's better we have longer unique names and then just `import vehicle_simulation_environment as sim`, or something along those lines.